data/data-pipeline/data_pipeline/score/field_names.py

```diff
@@ -219,6 +219,27 @@
     "Michigan EJSCREEN Priority Community"
 )
 
+# CDC SVI INDEX percentile fields
+CDC_SVI_INDEX_SE_THEME_PERCENTILE_FIELD: str = (
+    "Percentile ranking for Socioeconomic theme summary"
+)
+CDC_SVI_INDEX_HOUSEHOLD_THEME_PERCENTILE_COMPOSITION_FIELD: str = (
+    "Percentile ranking for Household Composition theme summary"
+)
+CDC_SVI_INDEX_LANGUAGE_THEME_PERCENTILE_FIELD: str = (
+    "Percentile ranking for Minority Status/Language theme summary"
+)
+CDC_SVI_INDEX_HOUSING_TRANSPORTATION_PERCENTILE_FIELD: str = (
+    "Percentile ranking for Housing Type/Transportation"
+)
+CDC_SVI_INDEX_RPL_THEMES_OVERALL_PERCENTILE_FIELD: str = (
+    "Overall percentile ranking for themes"
```

Comment:
Same comment as other PR: "themes" is too vague when it's combined with other fields. Could we do "Social Vulnerability Index (percentile)"?

Reply:
This is reflected in the documentation (page 4). I am not sure if this conveys the same intent as what is outlined by the authors. 
<img width="740" alt="Screen Shot 2022-01-13 at 5 24 24 PM" src="https://user-images.githubusercontent.com/94847739/149418582-ea6f00aa-23eb-49c7-a103-af3eddca6e20.png">

Reply:
The "theme" is actually what the authors call these fields. I can certainly add the "percentile" suffix though. What do you think?